hadoop/src/main/java/org/apache/carbondata/hadoop/api/CarbonInputFormat.java

```diff
@@ -150,6 +151,10 @@ public int getNumBlocks() {
     return numBlocks;
   }
 
+  public void setFileLists(List fileLists) {
```

Comment:
This change is not related to binary, suggest to keep it in separate PR

Reply:
ok, removed